tests/typ/compiler/ops-date.typ

```diff
@@ -0,0 +1,96 @@
+// negate Duration
+// Ref: false
+#test(-duration(hours: 2), duration(hours: -2))
+
+---
+// add and subtract Durations
+// Ref: false
```

Comment:
you won't need the individual `// Ref: false` lines if there's one at the top, _before_ the first `---`